docs/advanced.md

```diff
@@ -89,6 +89,9 @@ h2_response = h2_client.get("https://myserver.com")
 HTTPX supports .netrc file. In `trust_env=True` cases, if auth parameter is
 not defined, HTTPX tries to add auth into request's header from .netrc file.
 
+HTTPX has been loaded that content within the .netrc file is only checked once.
+The recommended way to refresh the cache would be to restart the interpreter.
```

Comment:
This needs updating now that we cache per-client; also suggesting putting this in an admonition:

```markdown
!!! note
    The NETRC file is cached across requests made by a client. If you need to refresh the cache (e.g. because the NETRC file has changed), you should create a new client or restart the interpreter.
```